packages/zoe/src/contractFacet/seat.js

```diff
@@ -15,13 +15,25 @@ export const makeZcfSeatAdminKit = (
   seatData,
   getAmountMath,
 ) => {
-  // The proposal and notifier are not reassigned.
-  const { proposal, notifier } = seatData;
+  // The notifier is not reassigned.
+  const { notifier } = seatData;
 
-  // The currentAllocation and exited may be reassigned.
+  // The proposal, currentAllocation and exited may be reassigned.
   let currentAllocation = harden(seatData.initialAllocation);
   let exited = false; // seat is "active"
 
+  // seats without a proposal shouldn't check offer safety.
+  let proposal;
+  let proposalSupplied;
+  if (seatData.proposal) {
+    proposalSupplied = true;
+    proposal = seatData.proposal;
+  } else {
+    proposalSupplied = false;
+    // enough to mollify typescript. uses should be guarded by !proposalSupplied
+    proposal = { give: null, want: null, exit: null };
```

Comment:
```suggestion
    proposalSupplied = false;
```
For whatever reason, it looks like typescript no longer needs this line